### PR TITLE
add a11y tutorial

### PR DESCRIPTION
**Pull request by SlicedSilver**
**Type of PR:** Adding a new tutorial to documentation site

**PR checklist:**

- [x] Documentation update

**Overview of change:**
Adds a tutorial for improving A11y on a simple LWC chart.

Preview: https://luxury-marshmallow-ad795c.netlify.app/lightweight-charts/tutorials

### CODE REVIEW

**Comment by anachuprina** [comment]
Minor:
- Inconsistent headings style (sentence case & title case)
- Inconsistent use of the words: JS/JavaScript
- The wording and grammar used in the conclusion are a bit complicated

File: website/tutorials/a11y/intro.mdx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+pagination_prev: null
+pagination_next: a11y/keyboard
+---
+
+# Improving Accessibility
+
+## Introduction
+
+This tutorial introduces how to make charts with Lightweight Charts™ more
+accessible. Lightweight Charts does not have built-in accessibility attributes
```
[the remark this answers is not in the document]
Comment (SlicedSilver) [reply website/tutorials/a11y/intro.mdx:24]:
```suggestion
accessible. Lightweight Charts™ does not have built-in accessibility attributes
```

File: website/tutorials/a11y/screenreader.mdx [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+[ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA)
+(Accessible Rich Internet Applications) roles and states offer a powerful
+toolkit to improve this interaction by conveying information about the behavior
+and purpose of interface components.
+
+In the context of our line chart, we'll be looking at how to implement ARIA
+properties to further enhance its accessibility in addition to generating
+descriptive content for our graph.
+
+## Use of ARIA attributes in the chart
```
[the remark this answers is not in the document]
Comment (SlicedSilver) [reply website/tutorials/a11y/screenreader.mdx:31]: There are more attributes which can be used, for this example we are only focusing on the most important attributes.

[the remark this answers is not in the document]
Comment (anachuprina) [reply website/tutorials/a11y/screenreader.mdx:31]: Maybe we can mention this and add a link to those other attributes?
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes

[the remark this answers is not in the document]
Comment (SlicedSilver) [reply website/tutorials/a11y/screenreader.mdx:40]: 👍 3d4b19402fe9cb72f6cb4958639c134ad3bb2b96

**Review by anachuprina — APPROVED** [review]

**Pull request merged**